fuzz-requirements.txt

```diff
@@ -0,0 +1,17 @@
+bitstring
```

Comment:
Protip you can use "-r filename.txt" to include another requirements file.

I suspect the top of this file could be replaced with "-r test-requirements.txt"

Reply:
AFL seems to take the test cases (provided in tests/in) and alters them in ways to produce errors. The mass amount of smaller files gives better results performance wise over one bigger file.